frontend/src/static/js/v_summary.js

```diff
@@ -188,7 +194,7 @@ window.vSummary = {
       }
 
       addHash('timeframe', this.filterTimeFrame);
-      addHash('mergegroup', this.isMergeGroup);
+      addHash('mergegroup', this.mergedGroups);
```

Comment:
The merged groups are not restored after refreshing

Reply:
Fixed.